### DISCUSSION TITLE
搜索框的图标可以来个点击事件吗

### DISCUSSION THREAD

**Discussion by weeptao** [ant-design/ant-design#41463] [Q&A]
![image](https://user-images.githubusercontent.com/103228833/227763640-b6a17383-fcb1-4c90-ad3d-1cbb289b1aae.png)

**Comment by kiner-tang** [comment]
这个你可以自己在图标增加点击事件呀
<img width="364" alt="image" src="https://user-images.githubusercontent.com/10286961/227764125-d39a3df8-65c7-4dc9-9f6d-da41a2f9aa87.png">

https://codesandbox.io/s/sou-suo-kuang-antd-5-3-2-forked-o2j1k6?file=/demo.tsx
